interface/main/calendar/add_edit_event.php

```diff
@@ -185,6 +200,17 @@ function DOBandEncounter()
              }
      }
     }
+    // auto create encounter fot therapy group
+    if(!empty($_POST['form_gid'])){
+                                                                                // status Took Place is the check in of therapy group
+        if ($GLOBALS['auto_create_new_encounters'] && $event_date == date('Y-m-d') && $_POST['form_apptstatus'] == '=') {
```

Comment:
I am a bit confused why the $_POST['form_apptstatus'] == '=' is here when this is meant for the group therapy. To clarify, is this item coming from the Appointment Status, Group Attendance Status, or the Group Status ?